molecule/verifier/testinfra.py

```diff
@@ -126,9 +126,18 @@ def bake(self):
 
         :return: None
         """
+        list_of_files_or_dirs = []
         options = self.options
         verbose_flag = util.verbose_flag(options)
-        args = verbose_flag + self.additional_files_or_dirs
+        additional_files_or_dirs = self.additional_files_or_dirs
+        for additional_path in additional_files_or_dirs:
+            glob_add_path_dir = glob.glob(self._config.scenario.directory + '/' + additional_path)
```

Comment:
In order to concatenate paths, the recommended way is to use `os.path.join` (or methods from Pathlib), since `/` is the POSIX separator, and if maybe will work on Windows, it's not the main separator there. [Reference](https://docs.python.org/3.6/library/os.html#os.sep).

Reply:
Indeed, I forgot about Windows. Will update it. Thanks!